applications/StructuralMechanicsApplication/custom_conditions/small_displacement_line_load_condition.h

```diff
@@ -44,14 +43,15 @@ namespace Kratos
 ///@{
 
 /**
- * @class LineLoadCondition2D
+ * @class SmallDisplacementLineLoadCondition
  * @ingroup StructuralMechanicsApplication
  * @brief This class is the responsible to add the contributions of the RHS and LHS of the line loads of the structure
  * @details It allows to consider different types of pressure and line loads
- * @author Riccardo Rossi
+ * @tparam TDim The dimension of the condition
  * @author Vicente Mataix Ferrandiz
  */
-class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) LineLoadCondition2D
+template<std::size_t TDim>
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) SmallDisplacementLineLoadCondition
```

Comment:
does not make sense to derive it from the non-small ones?

Reply:
Done